doc/compiling/linux.md

```diff
@@ -18,6 +18,7 @@
 | JsonCPP    | 1.0.0+  | Bundled JsonCPP is used if not present |
 | Curl       | 7.56.0+ | Optional   |
 | gettext    | -       | Optional   |
+| OpenSSL    | 3.0+    | Optional (only libcrypto used) |
```

Comment:
we should also install it for the linux CI runs
maybe keep one where it stays disabled to verify the fallback code

Reply:
Done. Seems to work.

(Only `gcc_14` is using it, because `libssl-dev` is an old version in `clang_7` and `gcc_7`.)